src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo-link"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              class="nav-link is-active"
+              href="#"
```

Comment:
The 'href' attribute is currently set to '#'. Consider replacing it with an actual URL or path to ensure proper navigation.